src/components/Login.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import TextError from "./common/TextError";
+import { Form, Formik, Field, ErrorMessage } from "formik";
+import * as yup from "yup";
+import { useNavigate } from "react-router-dom";
+
+interface loginProps {
+	email: "";
+	password: "";
+}
+export default function Login() {
+	const navigate = useNavigate();
+	const [records, setRecords] = useState<any[]>([]);
+
+	let valideSchema = yup.object({
+		email: yup.string().email("invalid").required("Required"),
+		password: yup.string().required("Required"),
+	});
+	const handleSubmit = (
+		values: loginProps,
```

Comment:
Use name Props on to pass that in props not on any place or just name it generic like login credentialType